doc/source/contributor-how-to-build-docker-images.rst

```diff
@@ -65,25 +65,26 @@ Building the base image
    * - ``FLWR_VERSION``
      - Version of Flower to be installed.
      - Yes
-     - ``1.9.0``
+     - :substitution-code:`|current_flwr_version|` 
    * - ``FLWR_PACKAGE``
      - The Flower package to be installed.
      - No
      - ``flwr`` or ``flwr-nightly``
 
 
 The following example creates a base Ubuntu/Alpine image with Python 3.11.0, pip 23.0.1,
-setuptools 69.0.2 and Flower 1.9.0:
+setuptools 69.0.2 and Flower |current_flwr_version|:
 
 .. code-block:: bash
+   :substitutions:
 
-  $ cd src/docker/base/<ubuntu|alpine>
-  $ docker build \
-    --build-arg PYTHON_VERSION=3.11.0 \
-    --build-arg FLWR_VERSION=1.9.0 \
-    --build-arg PIP_VERSION=23.0.1 \
-    --build-arg SETUPTOOLS_VERSION=69.0.2 \
-    -t flwr_base:0.1.0 .
+   $ cd src/docker/base/<ubuntu|alpine>
```

Comment:
this (and all the other) spaces are intentional?

Reply:
Yes, otherwise the `code_blocks` are invalid (because of the `:substitution:` argument)